infra/bootstrap.sh

```diff
@@ -98,13 +98,13 @@ if [[ ! -z "${RUN_BOOTSTRAP:-}" ]]; then
     "$SCRIPT_DIR"/sdk_helpers.sh ensure_registry "${REGISTRY_NAME_TOMORROW}"
     
     echo_title "Ensuring CPU compute"
-    "$SCRIPT_DIR"/sdk_helpers.sh ensure_aml_compute "cpu-cluster" 0 12 "Standard_DS3_v2"
+    "$SCRIPT_DIR"/sdk_helpers.sh ensure_aml_compute "cpu-cluster" 0 20 "Standard_DS3_v2"
     "$SCRIPT_DIR"/sdk_helpers.sh ensure_aml_compute "automl-cpu-cluster" 0 4 "Standard_DS3_v2"
     # Larger CPU cluster for Dask and Spark examples
     "$SCRIPT_DIR"/sdk_helpers.sh ensure_aml_compute "cpu-cluster-lg" 0 4 "Standard_DS15_v2"
     
     echo_title "Ensuring GPU compute"
-    "$SCRIPT_DIR"/sdk_helpers.sh ensure_aml_compute "gpu-cluster" 0 8 "Standard_NC12"
+    "$SCRIPT_DIR"/sdk_helpers.sh ensure_aml_compute "gpu-cluster" 0 20 "Standard_NC6"
```

Comment:
We may not have enough quota in the subscription to create 20 node GPU. Let's update node count from 8 to 12 for GPU.

Reply:
@harneetvirk This change reduces the size as well as increasing the maximum.  20 NC6 nodes is equivalent to 10 NC12 nodes.
Also, we are currently running with 21 NC6 nodes, which is 132 cores out of a quota of 250.